src/Symfony/Component/Console/Terminal.php

```diff
@@ -52,6 +52,15 @@ public function getHeight()
         return self::$height;
     }
 
+    /**
+     * Register terminal related environment variables.
+     */
+    public function registerEnv()
```

Comment:
Perhaps you're right. This would make this PR a bugfix only (despite the fact it can only target 3.2 right now), but I hesitated. I thought this method would be convenient to easily register all terminal related env vars at once in a script before calling a process, even if that's not much.

But fair enough, I'll remove it for now.